SETUP/tests/jsTests/formatPreviewTests.js

```diff
@@ -0,0 +1,113 @@
+/* global QUnit analyse */
+
+QUnit.test("Format preview test", function( assert ) {
+    let inLineTags = "[ibfg]|sc";
+    let suppress = {};
+    let text;
+    let issArray;
+
+    // assert start, length code and type correct for issue at index
+    function issueTest(index, start, length, code, type) {
+        let issue = issArray[index];
+        assert.deepEqual(issue.start, start);
```

Comment:
do we need deepEqual? Are these objects? Could we use strictEquals instead?

Reply:
Yes, I should have read the documentation more carefully.